pkg/utils/utils.go

```diff
@@ -230,16 +230,15 @@ func setSriovNumVfs(pciAddr string, numVfs int) error {
 
 func setNetdevMTU(pciAddr string, mtu int) error {
 	glog.V(2).Infof("setNetdevMTU(): set MTU for device %s", pciAddr)
-
-	ifaceName, err := dputils.GetNetNames(pciAddr)
-	if err != nil {
-		glog.Warningf("setNetdevMTU(): fail to get interface name for %s: %s", pciAddr, err)
-		return err
-	}
-	mtuFile := "net/" + ifaceName[0] + "/mtu"
-	mtuFilePath := filepath.Join(sysBusPciDevices, pciAddr, mtuFile)
 	b := backoff.NewConstantBackOff(1 * time.Second)
-	err = backoff.Retry(func() error {
+	err := backoff.Retry(func() error {
```

Comment:
could you explain why above lines are moved into `backoff.Retry()`?

Reply:
Sometimes, after VFs are created, the sysfs is not updated immediately. In that case, the `dputils.GetNetNames()` call will failed with `file/directory not exist`. Therefore I move it into the retry cycle.